webapp/src/pages/boardPage.tsx

```diff
@@ -66,6 +69,16 @@ const BoardPage = (props: Props): JSX.Element => {
         octoClient.workspaceId = workspaceId
     }, [match.params.workspaceId])
 
+    // Load user's block subscriptions when workspace changes
+    useEffect(() => {
+        // block subscriptions are relevant only in plugin mode.
+        if (!Utils.isFocalboardPlugin() || !me) {
+            return
+        }
+
+        dispatch(fetchUserBlockSubscriptions(me!.id))
+    }, [match.params.workspaceId])
```

Comment:
If this is only relevant for plugin mode, it should only be set if we're in plugin mode to save cycles:
```suggestion
    // block subscriptions are relevant only in plugin mode.
    if (Utils.isFocalboardPlugin()) {
        useEffect(() => {
            if (!me) {
                return
            }

            dispatch(fetchUserBlockSubscriptions(me!.id))
        }, [match.params.workspaceId])
    }
```